src/main/java/org/jenkinsci/plugins/ewm/definitions/Disk.java

```diff
@@ -24,16 +27,17 @@
  *
  * @author Alexandru Somai
  */
-public class Disk implements Describable<Disk> {
+public abstract class Disk extends AbstractDescribableImpl<Disk> {
```

Comment:
There is only one concern that I have about this approach, which is Backwards Compatibly. I never fully understood how to make sure it's not broken, but here is some useful info https://wiki.jenkins.io/display/JENKINS/Hint+on+retaining+backward+compatibility.

How can we make sure that we are backwards compatible with this approach?

Reply:
Well, I thought about this question before, and I think this would be the safe way to do it, as all old logic used only Disk's fields. I ll read that blog as well